src/App.js

```diff
@@ -1,13 +1,111 @@
-import React from 'react';
+import React from "react";
+import TodoList from "./components/TodoComponents/TodoList";
+import TodoForm from "./components/TodoComponents/TodoForm";
+import "./components/TodoComponents/Todo.css";
+
+const initialTodos = [
```

Comment:
Create a data.js file and put your data in there - then import it. It will make your code even cleaner :)